docs/administrator/deploy/infrastructure.md

```diff
@@ -10,7 +10,7 @@ Certain aspects of the infrastructure can be configured on an application by app
 * [Deploying an application with custom images](#deploying-an-application-with-custom-images)
 
 ## Configuring external Postgres instances
-By default, data for the query service, preference service, form service and runtime bundle is stored in an instance of Postgres deployed into an application namespace. A number of options are available to update the Postgres storage options: 
+By default, data for the application runtime bundle and application query service is stored in an instance of Postgres deployed into an application namespace. A number of options are available to update the Postgres storage options: 
```

Comment:
we are trying to give the runtime-bundle a different name: process-runtime
All depends in how many places runtime-bundle is already used i guess

Reply:
Ah ok. Let me merge this one and create an issue to update the name as it's used in most pages I think